src/code/templates/egg-game.js

```diff
@@ -13,6 +13,12 @@ import ChromosomeImageView from '../components/chromosome-image';
 import TimerSet from '../utilities/timer-set';
 import t from '../utilities/translate';
 
+// debugging options to shorten animation/randomization for debugging purposes
+// IMPORTANT: remember to set these back to their defaults (e.g. false, false, 0) before committing
+const debugSkipIntroGameteAnimation = false,  // set to true to skip intro gamete animation
+      debugSkipRandomGameteAnimation = false, // set to true to skip randomized gamete animation
+      debugTotalGameteCount = 0;              // non-zero value stops randomization after # gametes
```

Comment:
Nice :)

I can imagine that eventually we'll want a bunch of debugging cheat codes, which perhaps look up values on `window` so that we can set them directly from the command line. (If we ever do, though, we'll need to obfuscate them to prevent students from discovering them...)